crates/oxc_linter/src/rules/eslint/no_else_return.rs

```diff
@@ -224,25 +225,21 @@ fn no_else_return_diagnostic_fix(
         return;
     }
     ctx.diagnostic_with_fix(diagnostic, |fixer| {
-        let target_span = Span::new(prev_span.end, else_content_span.end);
+        // find the "else" word start
+        let else_word_start = ctx
+            .source_range(else_keyword_span)
+            .find("else")
+            .map_or(else_keyword_span.start, |pos| else_keyword_span.start + pos as u32);
+        let target_span = Span::new(else_word_start, else_content_span.end);
```

Comment:
```suggestion
        let target_span = Span::new(else_keyword_span.start, else_content_span.end);
```
Can we simplify the code to the above? I don't understand why we need to `+ pos`. And see the test changes, this will produce many unnecessary spaces, although it doesn't matter.

Reply:
Thank you for your review, i will simplify the code.
By the way, the unnecessary spaces are actually caused by use the `span` of `BlockStatement`, which aligns with ESLint's behavior.